test/suites/network/multi-nic.robot

```diff
@@ -173,14 +173,23 @@ Verify MicroShift On Single NIC
 
 Configure Subject Alternative Name
     [Documentation]    Replace subjectAltNames entries in the configuration
-    ...    to include only the one provided in ${ip}.
-    [Arguments]    ${ip}
-
-    ${subject_alt_names}=    CATENATE    SEPARATOR=\n
-    ...    ---
-    ...    apiServer:
-    ...    \ \ subjectAltNames:
-    ...    \ \ - ${ip}
+    ...    to include the IPs provided
+    [Arguments]    ${ip_1}    ${ip_2}
```

Comment:
Already told through Slack, just leaving here for reference. Could we try doing lists here? It would simplify both the keyword and the usage.